l10n_es_account_menu/__manifest__.py

```diff
@@ -1,17 +1,17 @@
-# Copyright NuoBiT - Kilian Niubo <kniubo@nuobit.com>
+# Copyright NuoBiT - Eric Antones <eantones@nuobit.com>
+# Copyright NuoBiT 2025 - Bijaya Kumal <bkumal@nuobit.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
 {
     "name": "L10n ES Account Menu",
     "summary": "This module extends account menu",
-    "version": "14.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Accounting",
-    "author": "NuoBiT Solutions, S.L.",
+    "author": "NuoBiT Solutions SL",
     "website": "https://github.com/nuobit/odoo-addons",
     "license": "AGPL-3",
-    "depends": ["account_menu", "l10n_es_extension"],
```

Comment:
This dependency is necessary for having the "is_vat" field, you cannot remove it. Migrate first https://github.com/nuobit/odoo-addons/tree/14.0/l10n_es_aeat_extension